streaming/README.md

```diff
@@ -511,4 +511,99 @@ metrics, following the same pattern of the receiving side.
 
 For a practical example see [Monitoring ephemeral nodes](#monitoring-ephemeral-nodes).
 
+## Troubleshooting streaming connections
+
+Sometimes when we are configuring master and slave nodes we can have some errors in our log files, in this section we are
+describing the most common errors that you can see.
+
+### Connection between slave and master is slow
+
+When we have a slow connection between master and slave, Netdata can raise different errors related to this problem,
+in the slave side we will have for the majority of the cases the error message
```

Comment:
I did not accept this completely, I changes slave system to slave Netdata, because Netdata is working like slave on the host, can we move with my suggestion?

Reply:
That works just fine, thanks.